src/h5cli/cli.py

```diff
@@ -31,7 +33,20 @@ def do_source(self, args):
         make_option('-m', '--mode', type=str, default='r', help='File mode ([r]ead, [w]rite, [a]ppend)')
     ])
     def do_load(self, args, opts=None):
+        """Load an hdf5 file.
+
+        Usage: load [-m mode] file [path]
+        """
+        path = '/'
+        if len(args) > 1:
+            path = args[1]
+        if len(args) == 0:
+            args = [self.explorer.filename]
+            path = self.explorer.working_dir
+            self.explorer.close()
+
         self.explorer = explorer.H5Explorer.from_file(args[0], mode=opts.mode)
+        self.explorer.change_dir(path)
```

Comment:
What's the argument for why we want to change to the path of the file that they are loading?

Reply:
This is the path within the file. 
This allows for reloading of the same file (#43) without moving. This is also used as the method of implementing `su` and `sudo` to keep you in the same placement. 

Lastly, while I was doing this, I added an optional argument to the load command which allows you to do `load foo.h5 /path/to/bar` without needing to cd as a separate step. This is not strictly speaking necessary, but is a mild convenience that was easy to add.

Reply:
ping @lphk92